handbook/technical-overview/feature-flags.md

```diff
@@ -6,7 +6,7 @@ If you want to introduce something behind a feature flag you can follow these st
 
 1. Ask someone from DevOps for invite to ConfigCat.
 2. Once you're in (https://app.configcat.com/) you can add your feature flag. The initial values should always be "On" in Dev and (probably always to start with) "Off" in Production and Staging.
-3. Make sure that the CONFIGCAT_SDK_KEY environment variable is in `.env.secrets` in the root of the repository. You can fetch it by calling `./scripts/secrets.sh configcat`.
+3. Make sure that the CONFIGCAT_SDK_KEY environment variable is in `.env.secret` in the root of the repository. You can fetch it by calling `yarn get-secrets configcat`.
```

Comment:
This should actually not fetch it from configcat. This should still fetch it from the project which uses configcat, e.g. `yarn get-secrets service-portal` should fetch the configcat sdk variable as well.